reinforcement_learning/rlclientlib/ranking_event.h

```diff
@@ -32,28 +41,39 @@ namespace reinforcement_learning {
     ranking_event();
     ranking_event(ranking_event&& other);
     ranking_event& operator=(ranking_event&& other);
+    ranking_event(utility::data_buffer& oss, const char* event_id, const char* context,
```

Comment:
Merge conflict: these public constructors were replaced with static ranking_event::choose_rank, outcome_event::report_action_taken, outcome_event::report_outcome and are not called anymore.

Reply:
I will double check this.

---
In reply to: [228609821](https://github.com/VowpalWabbit/vowpal_wabbit/pull/1653#discussion_r228609821) [](ancestors = 228609821)